docs/lib/datastore/fragments/android/datastore-events.md

```diff
@@ -0,0 +1,48 @@
+For instance, to listen to see if the network status is active, you could set up the following listener:
+
+```js
```

Comment:
Probably need to remove the JS sample from the android fragment, right?

Reply:
Fixed.